server/sonar-web/src/main/js/apps/coding-rules/rule/rule-issues-view.js

```diff
@@ -61,7 +61,7 @@ export default Marionette.ItemView.extend({
     return _.extend(Marionette.ItemView.prototype.serializeData.apply(this, arguments), {
       total: this.total,
       projects: this.projects,
-      baseSearchUrl: '/issues/search#resolved=false|rules=' + encodeURIComponent(this.model.id)
+      baseSearchUrl: window.baseUrl + '/issues/search#resolved=false|rules=' + encodeURIComponent(this.model.id)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert this concatenation to the use of a template. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3AS3512)